### ISSUE TITLE
[Fred + Collections] Optionset for pages does not "see" collections type resource

### ISSUE THREAD

**Issue by JANogueira** [modxcms/fred#281]
Hi All,

While using Modx 2.7.1-pl, Fred 1.1.0 RC and Collections 3.7.0 beta, you are not able to "see" it on Fred page property on the optionsets.

How to replicate?

Have a fred optionset having something like this:
```

{
      "name": "testemonials_parent_page",
      "label": "Listing Parent Page",
      "type": "page",
      "clearButton": true,
      "parents": "0",
      "depth": "1",
      "value": ""
    }

```

Then create a collections page so you can see and manage it on the fred frontend.
Then, if you go to the block having this property (to build listings based on Parentpage), the page is not shown on the dropdown box.

Thanks in advance.

**Comment by theboxer** [comment]
What page you don't see in the select box? The collection? If so, I can't replicate it, I can see collections resource there fine...

**Comment by JANogueira** [comment]
Hi John,

Yes it is the page that is the collection that has regular child resources (document).
Let me try to upgrade to the new Fred 1.1.0 rc2 and try to replicate again. I will share a video if I am able to replicate.

**Comment by JANogueira** [comment]
This is not replicable on Fred 1.1.0 RC2
I was able to replicate the issue using RC. Just upgraded to RC2 and the issue is not present anymore.

Thanks for your time John. Closing this.

**Comment by JANogueira** [comment]
Hi John,

I was able to replicate this problem with the new Fred 1.1.0 RC3.
To reproduce this, this is what you have to do:
- Have a collections type resource with children;
- Have an optionset with this config:
```
{
      "name": "testemonials_parent_page",
      "label": "Listing Parent Page",
      "type": "page",
      "clearButton": true,
      "parents": "0",
      "depth": "1",
      "value": ""
    }
```
- On the frontent, pick the Collection resource and save it.
- Refresh the page;
- Go again to the element settings, and click on the cross of the Resource selection box (as it uses `"clearButton": true,`) and you will see that the page disappears from the combo box.

If needed, I can show you this with a screencast.

Thanks

**Comment by JANogueira** [comment]
Well, this is embarrassing, but I created a new fresh setup due to other issue and tried to replicate it, and I simply can't again. But on the current website I am developing, it is still replicable. It is very strange...

I will try to compare any configuration differences.
The website that I am developing is multi context (with Babel and XRouting). The test environment with Fred Modx and Fred is not. Not sure if can be possibly the cause.

On the fresh setup I installed Fred 1.0.0-pl from the modx store, and then upgraded to 1.1.0 RC3. On the website I am facing the problem was upgraded to all releases since Fred Beta 2 as soon as these were made available.

I will share what I am able to find
